Core/include/Acts/Seeding/SpacePointGrid.hpp

```diff
@@ -37,8 +37,12 @@ struct SpacePointGridConfig {
   float cotThetaMax;
   // maximum impact parameter in mm
   float impactMax;
-  // sets of consecutive phi bins in the seed making step
-  int numPhiNeighbors = 0;
+  // number of phiBin neighbors (plus the current bin) that covers the full
+  // deflection of a minimum pT particle
+  int phiBinDeflectionCoverage = 1;
```

Comment:
@LuisFelipeCoelho, thanks for the name change!

You changed the calculation that uses phiBinDeflectionCoverage, which is fine, it needs to be carefully explained though: I suggest a comment like this: Multiplicator for the number of phi-bins. The minimum number of phi-bins depends on min_pt, magnetic field: 2*M_PI/(minPT particle phi-deflection). phiBinDeflectionCoverage is a multiplier for this number. If numPhiNeighbors (is this the correct naming? where is this configured?) is configured to return 1 neighbor on either side of the current phi-bin (and you want to cover the full phi-range of minPT), leave this at 3.

Reply:
I'd also set the default to 3.

Reply:
also, please run the seeding examples and check if they still work with the new config.

Reply:
Thank you very much for the additional explanation and for moving  :)

could you run the seeding example (not the itk one) to see if it still works? my expectation is that it won't work as it expects the numPhiNeighbors configuration (which now doesn't exist anymore)

Reply:
I tested the `seeding.py` and it worked. I think it is using the default value since there was no configuration for numPhiNeighbors in the seeding example. Should I add one configuration for numPhiNeighbors and phiBinDeflectionCoverage?